SSD/SSDDevice_test.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include "gmock/gmock.h"
+#include "SSDDevice.h"
+
+using namespace testing;
+using namespace std;
+
+class SSDDeviceFixture : public Test {
+public:
+
+};
+
+TEST_F(SSDDeviceFixture, ssdCreationTC) {
+    SSD ssd;
```

Comment:
이게 아래와 수정되어야 할것 같습니다.
```suggestion
    SSDDevice ssd;
```